ui-src/src/reducers/index.js

```diff
@@ -1,10 +1,11 @@
-import * as A from '../actions'
+import * as A from "../actions";
 
 const initialState = {
   // any app properties received from clutter backend
   appProperties: {},
   // posts with 'stamp' as their key
   posts: {},
+  isOpen: true,
```

Comment:
using a slightly more descriptive var name is a bit better. What isOpen? modalIsOpen